Shared/Samples/Show geodesic sector and ellipse/ShowGeodesicSectorAndEllipseView.swift

```diff
@@ -0,0 +1,338 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowGeodesicSectorAndEllipseView: View {
+    /// The data model that helps determine the view.
+    @StateObject private var model = Model()
+    
+    /// Manages the presentation state of the menu.
+    @State private var isPresented: Bool = false
+    
+    var body: some View {
+        MapViewReader { proxy in
```

Comment:
One more thing I forgot to mention: the proxy is a proxy for the map view itself. The name `proxy` is ambiguous. Proxy for what? Instead, consider using `mapView` as the name of the proxy:

```swift
MapViewReader { mapView in
    ...
}
```

Reply:
updated to 

`MapViewReader { mapView in`

 I'll use this going forward as well.